include/osquery/status.h

```diff
@@ -124,6 +127,16 @@ class Status {
     return ok();
   }
 
+  static Status success() {
+    return Status(success_code, std::string{"ok"});
```

Comment:
I'm not sure we need the `"ok"`. Also it is inconsistent with the ctro above which uses `"OK"`. Maybe remove both?

Reply:
Ok, you are right. I'll remove it.